src/node/rpc/memberfrontend.h

```diff
@@ -1,10 +1,11 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the Apache 2.0 License.
 #pragma once
+#include "../../luainterp/txscriptrunner.h"
```

Comment:
Minor: better to stick to the old include paths?

Reply:
This only worked since we have defined our src directory as a global include path.
Is there a reason to use the old paths?